src/parsers/transactions.ts

```diff
@@ -114,11 +113,10 @@ export const parseTransactions = async (
 
   const dedupedResults = uniqBy(updatedResults, txn => txn.hash);
 
-  const orderedDedupedResults = orderBy(
-    dedupedResults,
-    ['minedAt', 'nonce'],
-    ['desc', 'desc']
-  );
+  const orderedDedupedResults = dedupedResults.sort((a, b) => {
+    // Sort by `minedAt` in descending order and by `nonce` in descending order.
+    return b.minedAt! - a.minedAt!;
+  });
```

Comment:
Is it sorting by nonce tho?

Algorithmically achieving sort by multiple keys means sorting by each key in reverse order so `.sort(byNonce).sort(byMinedAt)` (more or less).